client/src/app/_components/groups/group-posts-list/index.tsx

```diff
@@ -43,15 +43,23 @@ const GroupPostsList = ({ groupId, refreshPosts, onRefresh }: GroupPostsListProp
 	}, [error]);
 
 	return (
-		<div className="flex flex-auto w-1/3 overflow-hidden">
+		<div className="flex flex-auto w-1/3 overflow-auto">
```

Comment:
I think the platform pages aren't meant to be scrollable (i.e. everything in one view without having to scroll down) so this should really be the tailwind equivalent of `overflow-y: scroll`. Feel free to change it but I'll probably do it myself anyway as I work through the wireframes

Reply:
All good brother, I'll leave the wireframe design to you.